### DISCUSSION TITLE
A clean interface option?

### DISCUSSION THREAD

**Discussion by scotttheis** [Comfy-Org/ComfyUI#5223] [Ideas]
Has anyone considered making an interface that is more user friendly? Or maybe made a clean front end that could be used with this?

**Comment by LukeG89** [comment]
I've never tried it, but there is [SwarmUI](https://github.com/mcmonkeyprojects/SwarmUI)

**Comment by LukeG89** [comment]
You can use [cg-controller](https://github.com/chrisgoringe/cg-controller) to create a simple interface over your workflow!
